handlers/analytics.go

```diff
@@ -191,8 +195,10 @@ func getOrAddMissing(key string, headers http.Header, missingHeaders []string) (
 	return "", missingHeaders
 }
 
-func toAnalyticsData(log *AnalyticsLog, geo AnalyticsGeo, extData analytics.ExternalData) []analytics.LogData {
+func (c *AnalyticsHandlersCollection) toAnalyticsData(log *AnalyticsLog, geo AnalyticsGeo, extData analytics.ExternalData) []analytics.LogData {
 	ua := useragent.Parse(log.UserAgent)
```

Comment:
Do they both contain different fields (i.e so we can't just completely switch to the new parser)?

Reply:
Unfortunately, they provide different fields for us 😠